java/Anton-Kozyk/src/com/zoolatech/lecture3/tasks/_1/MusicalInstruments.java

```diff
@@ -0,0 +1,40 @@
+package com.zoolatech.lecture3.tasks._1;
+
+abstract public class MusicalInstruments {
+    String instrumentName;
+    abstract void playSound();
+}
+
+class Guitar extends MusicalInstruments {
+    Guitar() {
+        instrumentName = "Guitar";
```

Comment:
it would be better to use a constructor of a parent class (since the variable `instrumentName` is defined there)